formatter/set.go

```diff
@@ -1,10 +1,20 @@
 package formatter
 
-import "github.com/sirupsen/logrus"
+import (
+	"os"
+
+	"github.com/sirupsen/logrus"
+)
 
 // SetTextFormatter set the text formatter for given logger.
 func SetTextFormatter(logger *logrus.Logger) {
-	logger.Formatter = NewTextFormatter()
+	var formatter logrus.Formatter
+	if os.Getenv("NB_LOG_FORMAT") == "json" {
```

Comment:
The logic does not match with the function name. If it is a text formatter then do not provide json formatter.
Please create a new SetJsonFormatter and call the proper function based on env in the util/log.go

Reply:
Good point, I forgot about that.